cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -558,6 +558,7 @@ def _create_block(request):
             "locator": str(created_xblock.location),
             "courseKey": str(created_xblock.location.course_key),
             "static_file_notices": asdict(notices),
+            "upstreamRef": str(created_xblock.upstream),
```

Comment:
Hmm, seems like we have been inconsistent about snake_case vs. camelCase in this JSON response.

Reply:
Yes, majorly camelCase at least in this file.

Reply:
Hmm, generally we use snake case for JSON though.